Description: Remove dist-lzma from AM_INIT_AUTOMAKE.
 Avoids "support for lzma-compressed distribution archives has been removed"
 error.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2015-03-08

--- a/configure.in
+++ b/configure.in
@@ -69,7 +69,7 @@
 dnl  Note that there's a copy of these options in the top-level Makefile.am,
 dnl  so update there too if changing anything.
 dnl
-AM_INIT_AUTOMAKE([1.11 gnu no-dependencies parallel-tests dist-bzip2 no-dist-gzip dist-lzma])
+AM_INIT_AUTOMAKE([1.11 gnu no-dependencies parallel-tests dist-bzip2 no-dist-gzip])
 AM_CONFIG_HEADER(config.h:config.in)
 AM_MAINTAINER_MODE
 
